src/layouts/Settings/SocialMediaSettings.tsx

```diff
@@ -69,7 +69,11 @@ const ValidatedFormInput = ({
   const displayedLabel = upperFirst(label)
   const { register } = useFormContext()
   const { errors } = useFormState()
+
+  // TODO: Marking this any due to type conflicts
   const isInvalid =
+    /* eslint-disable-next-line @typescript-eslint/ban-ts-comment */
+    // @ts-ignore implicit any type error
```

Comment:
hmm i can take a look at this

Reply:
@seaerchin I doubt this type error is because of any package mismatch right?

Reply:
oh fk the library exposes the `any`

Reply:
i don't get any type errors though weirdly (cos i should)

Reply:
@seaerchin i think when developing is fine, but if you run the build command it will fail